### DISCUSSION TITLE
New Tesla API for authentication

### DISCUSSION THREAD

**Discussion by inigoml** [teslamate-org/teslamate#3696] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

Tesla opened officially its API for third party applications months ago. Those apps that use API for retrieving data should migrate to use this new way to interact with API. Both Tessie, Teslafi and others have already migrated.

https://www.tesla.com/developer-docs

### Expected Behavior

Use new way to authenticate against Tesla API before access is broken.

### Steps To Reproduce

No steps

### Relevant log output

```shell
No log (yet)
```

### Type of installation

Docker

### Version

v1.28.3

**Comment by cwanja** [comment]
We are well aware of it and monitoring it. However, the way that Tesla is pushing right now it is not accessible for individual installations like TeslaMate. I am going to move this to a discussion to retain the conversation, however there is not an "issue" today.
